content/docs/week01.adoc

```diff
@@ -19,12 +19,13 @@ test each and every part manually. You clicked on the nice TestMyCode (TMC) butt
 There is a catch though: Out there in the real world, there won't be any button doing that magic for you, nor will some teacher or school
 provide the tests for you, so [red]*you will be on your own*.#
 
-But fret not! Writing tests is typically much simpler than writing the actual code. At least, it is when you follow a basic set of steps:
+But fret not! Writing tests is typically much simpler than writing the actual code. At least it is, when you know the anatomy of a test:
```

Comment:
AAA is more of an organisation-thingy.
In TDD, a helpful trick to learn can be to start with thinking of the 'Assert' part.
I'd therefore avoid thinking of them in steps.